gc/base/MemoryManager.cpp

```diff
@@ -508,6 +521,16 @@ MM_MemoryManager::destroyVirtualMemory(MM_EnvironmentBase* env, MM_MemoryHandle*
 		}
 	}
 
+#if defined(OMR_ENV_DATA64) && !defined(OMR_GC_COMPRESSED_POINTERS)
+	MM_GCExtensionsBase* extensions = env->getExtensions();
+	MM_VirtualMemory* shadowMemory = extensions->shadowHeapHandle->getVirtualMemory();
```

Comment:
also needs a guard against shadowHeapHandle being NULL, otherwise it will cause crashes on shutdown when verifier is not enabled.